server/liveblog/syndication/syndication.py

```diff
@@ -33,11 +33,13 @@
         'type': 'objectid',
         'required': True
     },
-    'last_delivered_post_id': Resource.rel('posts', embeddable=True, required=True,
-                                           nullable=True, type="objectid"),
     'token': {
         'type': 'string',
         'unique': True
+    },
+    'start_date': {
+        'type': 'datetime',
+        'default': None
```

Comment:
Unfortunately python-eve doesn't set the default value I have selected (None) and set by default to datetime.datetime.now(). I am investigating about this issue.